config.entry.json

```diff
@@ -1,7 +1,13 @@
 {
   "Services": {
     "httpproxy": {
-      "maxPrice": "0.001"
+      "maxPrice": "0.001",
+      "IPFilter": {
+        "Allow": [
```

Comment:
As we discussed, we need to add a Location with empty country in default config so user will know what to add